reference-files/plugin.sketchplugin/Contents/Sketch/shapes.js

```diff
@@ -26,7 +26,7 @@ function main(ctx) {
   })
 
   // Save and close
-  doc.save(ctx.pwd + '/output.sketch', {
+  doc.save(String(ctx.savePath), {
     saveMode: Document.SaveMode.SaveAs,
   })
   doc.close()
```

Comment:
I missed that earlier but the `.close()` should be in the callback of the `.save` method

Reply:
Ah ok, yes makes sense.